### ISSUE TITLE
Confirmed working for DS1019+

### ISSUE THREAD

**Issue by angeloalduino** [007revad/Synology_enable_M2_volume#53]
DS1019+ on DSM 7.2-64561
2x WD_BLACK 1TB SN770 m.2 using RAID 1

Question: The docs say `After any DSM update you will need to run this script, and the Synology_HDD_db script again.`. Can you clarify what will happen if you don't run the scripts again, and when exactly the scripts should be run (before/after shutdown)? If the scripts are not run and the system reboots with a new version, could it cause data loss? Or would we simply run the scripts again and then the volumes would reappear?

Thank you!

**Comment by 007revad** [comment]
I'd schedule both to run at boot-up in the same scheduled task. But you can run them via SSH if you prefer. You may have to then reboot once more before you see your M2 volume in storage manager.

**Comment by angeloalduino** [comment]
Thanks @007revad! Would be great to make that clear in the docs so others can be more at ease 👍 Thank you so much for your work!

**Comment by Fonta** [comment]
Also works on 1019+ - DSM 7.2-64570 Update 1

**Comment by angeloalduino** [comment]
I had the script scheduled to run at shutdown and then installed `7.2-64570 Update 1`. I'm not sure if it ran but when it came back up, the volume critical warning beep was repeating and it said the cache volume was invalid. I ran the script manually, rebooted again, and then everything came up normally.

I'm not sure whether the script is running properly at shutdown but I now enabled the 'Save output results' option in Task Manager to debug this in the future.

In summary: for anyone wondering what happens after a DSM update, it seems scary at first but everything comes back to normal after running the script again 👍

**Comment by 007revad** [comment]
Thanks @angeloalduino  

After DSM 7.2 was released I changed the readme for my [Synology_HDD_db](https://github.com/007revad/Synology_HDD_db) script to say to schedule it to run at boot-up, instead of shutdown, but forgot to change the readme for this script. I have changed it now.

**Comment by Oujiii** [comment]
Hi @angeloalduino and @Fonta is still working for you? #168 mentions that this is only working for one drive and this is also on the README now.

Thanks.
